apps/media/src/app/app.service.ts

```diff
@@ -0,0 +1,129 @@
+import {Injectable} from '@nestjs/common'
+import {StorageClient} from '@wepublish/media/api'
+import sharp from 'sharp'
+import {TransformationsDto} from './transformations.dto'
+import {Readable} from 'stream'
+
+const getTransformationKey = (transformations: TransformationsDto) => {
```

Comment:
sorts the object keys and turns it into json. Sorting is needed so we don't have the same transformations multiple time just with a different sorting.